utils.py

```diff
@@ -453,3 +454,17 @@ def get_all_language_codes_and_names():
 def unescape_encoded_uri_component(escaped_string):
     """Unescape a string that is encoded with encodeURIComponent."""
     return urllib.unquote(escaped_string).decode('utf-8')
+
+
+cache_slug = None  # pylint: disable=invalid-name
+def get_cache_slug():
```

Comment:
There's an issue: I am currently caching the cache_slug, let's say feconf.DEV = True and the cache_slug is generated. Then, I swap it with feconf.DEV = False and request the cache slug again, and it returns the old value which is wrong.